manifests/i/IrfanSkiljan/IrfanView-plugins/4.58/IrfanSkiljan.IrfanView-plugins.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IrfanSkiljan.IrfanView-plugins
+PackageVersion: 4.58
+PackageLocale: en-US
+Publisher: Irfan Skiljan
+PublisherUrl: "https://www.irfanview.com/"
+PublisherSupportUrl: https://irfanview-forum.de
+PrivacyUrl: "https://www.irfanview.com/privacy_policy.htm"
+Author: Irfan Skiljan
+PackageName: IrfanView plugins
+PackageUrl: "https://www.irfanview.com/plugins.htm"
+License: Proprietary, free for non-commercial use
+LicenseUrl: "https://www.irfanview.com/eula.htm"
+Copyright: Copyright (C) 2021 by Irfan Skiljan, Wiener Neustadt, Austria.
+CopyrightUrl: https://www.irfanview.com/eula.htm
+ShortDescription: IrfanView plugins installer
```

Comment:
```
ShortDescription: IrfanView plugins installer
```
Write a better description.

Reply:
Sure, though I'm a bit confused. The manifest says the ShortDescription is what's used by `winget show`, and Description is never used. That doesn't seem true for me. `winget show irfanview` shows me the Description of IrfanView, and I can't find a command that shows the ShortDescription.

Reply:
> Sure, though I'm a bit confused. The manifest says the ShortDescription is what's used by `winget show`, and Description is never used. That doesn't seem true for me. `winget show irfanview` shows me the Description of IrfanView, and I can't find a command that shows the ShortDescription.

This is true if Description have Metadata it will show that when you do `winget show`, i am not completely sure where it will show the ShortDescription, but i believe its used by websites and different API's (winstall.app for example)